static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js

```diff
@@ -0,0 +1,189 @@
+            products,
+        },
+        {
+            id: 'remote',
+            products,
+            // eslint-disable-next-line import/no-shadow
+            test: (html: string, variant: EpicVariant, test: EpicABTest) => {
+                const ophan = config.get('ophan');
+                const page = config.get('page');
+
+                const tracking = {
+                    ophanPageId: ophan.pageViewId,
+                    ophanComponentId: 'ACQUISITIONS_EPIC',
+                    platformId: 'GUARDIAN_WEB',
+                    campaignCode: campaignId,
+                    abTestName: 'remote_epic_test',
+                    abTestVariant: 'api',
+                    referrerUrl:
+                        window.location.origin + window.location.pathname,
+                };
+
+                const localisation = {
+                    countryCode: 'US',
+                };
+
+                const targeting = {
+                    contentType: page.contentType,
+                    sectionName: page.sectionName,
+                    shouldHideReaderRevenue: page.shouldHideReaderRevenue,
+                    isMinuteArticle: page.isLiveBlog, // Is this the same thing?
+                    isPaidContent: page.isPaidContent,
+                    tags: buildKeywordTags(page),
+                };
+
+                const payload = JSON.stringify({
+                    tracking,
+                    localisation,
+                    targeting,
+                });
+
+                const trackingCampaignId = `epic_${campaignId}`; // note exposed on ABTest unfortunately
+
+                emitBeginEvent(trackingCampaignId);
+                console.log('epic - beginning remote variant...');
+
+                setupClickHandling(
+                    test,
+                    variant.campaignCode,
+                    products,
+                    variant.id
+                );
+
+                fetchRemoteEpic(payload)
+                    .then(checkResponseOk)
+                    .then(decodeJson)
+                    .then(json => {
+                        const epicHtml = json.html;
+                        const css = json.css;
+                        const content = `<style>${css}</style>${epicHtml}`;
+
+                        return fastdom.write(() => {
+                            const target = document.querySelector('.submeta');
+
+                            if (!target) {
+                                return;
+                            }
+
+                            const parent = target.parentNode;
+
+                            if (!parent) {
+                                return;
+                            }
+
+                            const container = document.createElement('div');
+                            parent.insertBefore(container, target);
+
+                            // use Shadow Dom if found
+                            if (container.attachShadow) {
```

Comment:
Given that we're now explicitly checking for availability of the Shadow DOM API (and falling back if not) I wonder if we should remove the polyfill. My experience in IE11 was that the polyfill makes `.attackShadow` available, but doesn't provide the isolation you'd expect (page styles were still bleeding into the epic).